distribution/docs/src/main/resources/_contents/_using/using-catalog-search-ui-contents.adoc

```diff
@@ -0,0 +1 @@
+placeholder text
```

Comment:
Can we add ticket numbers to help track these placeholders?  Do we expect to have them complete before DDF 2.10 is released in the next month or two?